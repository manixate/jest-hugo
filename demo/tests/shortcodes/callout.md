<test name="should render successful with alert">
  {{% callout type="alert" %}}
  An alert type callout
  {{% /callout %}}
</test>

<test name="should render successful with warning">
  {{% callout type="warning" %}}
  An warning type callout
  {{% /callout %}}
</test>

<test name="should show error when missing type parameter.">
  {{% callout %}}
  An note type callout
  {{% /callout %}}
</test>

<test name="should show error when enter invalid type.">
  {{% callout type="invalid" %}}
  An note type callout
  {{% /callout %}}
</test>