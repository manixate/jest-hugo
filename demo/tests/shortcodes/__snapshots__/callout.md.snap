// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`callout should render successful with alert 1`] = `
<div>
  <div class="callout callout-alert"
    role="callout">
    <div class="callout-header">Alert:
    </div>
    <div class="callout-content">An alert type callout</div>
  </div>
</div>
`;

exports[`callout should render successful with warning 1`] = `
<div>
  <div class="callout callout-warning"
    role="callout">
    <div class="callout-header">Warning:
    </div>
    <div class="callout-content">An warning type callout</div>
  </div>
</div>
`;

exports[`callout should show error when enter invalid type. 1`] = `
<div>
  shortcodes/callout.md: Invalid callout's type 'invalid'. Expecting one of 'alert warning'
</div>
`;

exports[`callout should show error when missing type parameter. 1`] = `
<div>
  shortcodes/callout.md: Invalid callout's type '%!s(<nil>)'. Expecting one of 'alert warning'
</div>
`;
